sharding/collation.go

```diff
@@ -5,11 +5,14 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto/sha3"
+	"github.com/ethereum/go-ethereum/rlp"
 )
 
 // Collation base struct.
 type Collation struct {
 	header       *CollationHeader
+	body         []byte
 	transactions []*types.Transaction
```

Comment:
+1. How are we handling a body and transactions? 

Are we supposed to update the body every time we modify this array?

Reply:
transactions are actual deserialized tx's that will be helpful for our implementation